assets/xml/scenes/dungeons/jyasinboss.xml

```diff
@@ -1,5 +1,6 @@
 <Root>
     <File Name="jyasinboss_scene" Segment="2">
+        <Cutscene Name="gNabooruKnuckleIntroCs" Offset="0x2BB0"/>
         <Cutscene Name="gNabooruKnuckleDefeatCs" Offset="0x3F80"/>
```

Comment:
should this also have a prefix like `SpiritBoss` or something (related to the scene)